development/bot/.env

```diff
@@ -0,0 +1,11 @@
+TELEGRAM_TOKEN=1923165985:AAHTOrkw68dZ1KsbqlrowwHWQ_QexfQdGng
+VK_TOKEN=d57a63108714f9808416cd877b08238739263fa7e0aeeaff198dc9e98976a3a587b70ea2bb786dce2b2f8
+BOT_NAME=TEST_NAME
+
+GREETING_MESSAGE=Привет, я Бот
+MOOD_MESSAGE=Всё плохо, меня заперли в этой коробке и заставляют разговаривать с людьми
+FAREWELL_MESSAGE=Приходи ещё
+ISUCT_MESSAGE=Диплом химтеха - залог успеха!
+COMPLIMENT_MESSAGE=Спасибо, мне очень приятно :)
+CONDOLENCES_MESSAGE=Я постараюсь выбраться из этого :(
+DIFFERENT_MESSAGE=Я вас не понимаю :(
```

Comment:
While not really a good way to store all bot responses' in `.env` file, it's ok for now.

But I would advise to store messages in quotation marks (" ") to avoid possible issues with string segmentation later.
For example like this:
```
SOME_MESSAGE_VARIABLE = "Your message"
```

Reply:
I delete it on later stage :(